### DISCUSSION TITLE
Email deliverability

### DISCUSSION THREAD

**Discussion by j2is** [swellstores/community#123] [Help] [answered]
I have a welcome email notification turned on, sometimes the email doesn't get sent out.
Is there a log somewhere to see what happened? 
If the email sent from Swell wasn't delivered successfully, it would be nice to retry, or send via another service – how would I go about that? Thanks so much

**Comment by ericingram** [comment, marked as answer]
We use Sendgrid on the backend to deliver emails. It's a very reliable system. We'll consider adding delivery events to the API at some point, but without that, you can check the `/notifications` endpoint to make sure the email was sent from Swell.

If you want more delivery details, we recommend enabling the Sendgrid integration with your own API key (Dashboard > Integrations).
